src/analyses/libraryFunctions.ml

```diff
@@ -128,6 +146,12 @@ let glibc_desc_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("__overflow", unknown [drop "f" [r]; drop "ch" []]);
     ("__ctype_get_mb_cur_max", unknown []);
     ("__xmknod", unknown [drop "ver" []; drop "path" [r]; drop "mode" []; drop "dev" [r; w]]);
+    ("__bad_percpu_size",  special' [] @@ fun () -> Abort); (* these do not have defintions so the linker will fail if they are actually called *)
+    ("__bad_size_call_parameter",  special' [] @@ fun () -> Abort);
+    ("__xchg_wrong_size",  special' [] @@ fun () -> Abort);
+    ("__cmpxchg_wrong_size",  special' [] @@ fun () -> Abort);
+    ("__xadd_wrong_size",  special' [] @@ fun () -> Abort);
+    ("__put_user_bad",  special' [] @@ fun () -> Abort);
```

Comment:
These all seem to be part of Linux kernel implementation, not glibc, so they should be in the following list. Since these lists are currently conditional, they will work for SV-COMP there as well, although these `abort()` implementations really should be in sv-benchmarks.

Also, there's no need for the functional form, could just be `special [] Abort`.